rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -558,21 +562,21 @@ public int listenPort() {
         return configFromFiles.getInt("peer.listen.port");
     }
 
-    public String getPeerDiscoveryBindAddress() {
-        if (bindIp != null) {
-            return bindIp;
+    public InetAddress getBindAddress() {
+        if (!configFromFiles.hasPath(PROPERTY_BIND_ADDRESS)) {
+            return InetAddress.getLoopbackAddress();
```

Comment:
For wire protocol and peer discovery use `DEFAULT_BIND_ADDRESS` if it is not defined.